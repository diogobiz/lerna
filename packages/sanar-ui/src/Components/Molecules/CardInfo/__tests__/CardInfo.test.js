
import React from 'react'
import TestRenderer from 'react-test-renderer'

import ESCardInfo from '../'

it('renders correctly', () => {
    const component = (
        <ESCardInfo>
            <h1>ESCardInfo</h1>
        </ESCardInfo>
    )

    const tree = TestRenderer.create(component).toJSON()
    expect(tree).toMatchSnapshot()
})
