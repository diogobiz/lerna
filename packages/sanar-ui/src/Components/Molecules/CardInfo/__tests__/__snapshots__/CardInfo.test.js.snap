// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="es-card-info"
>
  <img
    alt=""
    width={52}
  />
  <div
    className="es-card-info__content"
  >
    <span
      className="es-badge es-badge__warning es-badge__border es-badge__warning--solid ant-badge ant-badge-not-a-wrapper"
    >
      <sup
        className="ant-scroll-number ant-badge-count"
        data-show={true}
        title="1"
      >
        <span
          className="ant-scroll-number-only"
          style={
            Object {
              "WebkitTransform": "translateY(-1100%)",
              "msTransform": "translateY(-1100%)",
              "transform": "translateY(-1100%)",
              "transition": "none",
            }
          }
        >
          <p
            className=""
          >
            0
          </p>
          <p
            className=""
          >
            1
          </p>
          <p
            className=""
          >
            2
          </p>
          <p
            className=""
          >
            3
          </p>
          <p
            className=""
          >
            4
          </p>
          <p
            className=""
          >
            5
          </p>
          <p
            className=""
          >
            6
          </p>
          <p
            className=""
          >
            7
          </p>
          <p
            className=""
          >
            8
          </p>
          <p
            className=""
          >
            9
          </p>
          <p
            className=""
          >
            0
          </p>
          <p
            className="current"
          >
            1
          </p>
          <p
            className=""
          >
            2
          </p>
          <p
            className=""
          >
            3
          </p>
          <p
            className=""
          >
            4
          </p>
          <p
            className=""
          >
            5
          </p>
          <p
            className=""
          >
            6
          </p>
          <p
            className=""
          >
            7
          </p>
          <p
            className=""
          >
            8
          </p>
          <p
            className=""
          >
            9
          </p>
          <p
            className=""
          >
            0
          </p>
          <p
            className=""
          >
            1
          </p>
          <p
            className=""
          >
            2
          </p>
          <p
            className=""
          >
            3
          </p>
          <p
            className=""
          >
            4
          </p>
          <p
            className=""
          >
            5
          </p>
          <p
            className=""
          >
            6
          </p>
          <p
            className=""
          >
            7
          </p>
          <p
            className=""
          >
            8
          </p>
          <p
            className=""
          >
            9
          </p>
        </span>
      </sup>
    </span>
    <div
      className="es-typography es-typography--caption es-typography--ellipsis"
      data-testid="es-typography"
    />
  </div>
</div>
`;
